Riot/Assets/en.lproj/Vector.strings

```diff
@@ -1027,8 +1027,8 @@
 
 // Setup
 
-"key_backup_setup_banner_title" = "Never lose encrypted messages";
-"key_backup_setup_banner_subtitle" = "Start using Key Backup";
+"key_backup_setup_banner_title" = "Secure Backup";
```

Comment:
We must use new string ids. This is different feature that needs the right translation in every language.